content/nginxaas-azure/app-protect/configure-waf.md

```diff
@@ -101,7 +101,28 @@ The following table shows the path to the precompiled policy file that needs to
 
 To view the contents of the available security policies, navigate to the azure portal and select the **Security Policies** tab in the App Protect section.
 
-{{<note>}}Custom policies are not supported at this time.{{</note>}}
+## Custom Policies
+
+NGINXaas for Azure also supports custom security policies. You can create and modify custom security policies to deploy to NGINX App Protect Instances using the API or Azure Portal.
+
+### Manage Custom Policies
+
+To create a custom security policy in the Azure Portal:
```

Comment:
Might want a section on deleting a custom policy and mention that a policy in use cannot be deleted.  Also, somewhere you should mention that the two columns are that show up on the custom policy panel: **Compilation**,  **Applied?**

Reply:
Done